backend/tests/integration/matcher/test_tasks.py

```diff
@@ -24,10 +24,18 @@ def test_match_references_by_uuids(inspire_app):
     }
     citer_record = create_record("lit", data=citer_data)
     excluded_citer_record = create_record("lit", data=citer_data)  # won't be passed
-
+    deleted_record = create_record(
+        "lit", data={"deleted": True}, with_control_number=True
+    )
     record_without_references = create_record("lit")
 
-    match_references_by_uuids([str(citer_record.id), str(record_without_references.id)])
+    match_references_by_uuids(
+        [
+            str(citer_record.id),
+            str(record_without_references.id),
+            str(deleted_record.id),
```

Comment:
what are you asserting here?

Reply:
not ready yet :) I just pushed.. I was planning to put some refs to it and assert that they aren't linked perhaps.